### PR TITLE
add missing default property to cdap-defaults.xml

### PR DESCRIPTION
**Pull request by albertshau**
Added missing property for active runtime monitoring.

### CODE REVIEW

**Review by CuriousVini — COMMENTED** [review]
this means there is a bug in `DefaultRuntimeJob#run` method where properties from RuntimeJobEnvironment are not being set correctly

**Review by CuriousVini — APPROVED** [review]
lgtm to unblock others. Will open followup PR to this.

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -529,6 +529,14 @@
     </description>
   </property>
 
+  <property>
+    <name>app.program.runtime.monitor.active.monitoring</name>
```
Comment (chtyim) [inline cdap-common/src/main/resources/cdap-default.xml]: This properties is only temporary until we switch to use the new RuntimeClient, which will be in in a day.

**Comment by albertshau** [comment]
I'm just going to change the getBoolean() call to have a default then. It can be removed by your future changes

File: cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/AbstractProgramTwillRunnable.java [hunk 2]
```diff
@@ -198,7 +198,7 @@ private void doInitialize(File programOptionFile) throws Exception {
     Injector injector = Guice.createInjector(createModule(cConf, hConf, programOptions, programRunId));
 
     // Setup the proxy selector for in active monitoring mode
-    if (cConf.getBoolean(io.cdap.cdap.common.conf.Constants.RuntimeMonitor.ACTIVE_MONITORING)) {
+    if (cConf.getBoolean(io.cdap.cdap.common.conf.Constants.RuntimeMonitor.ACTIVE_MONITORING, false)) {
```
Comment (CuriousVini) [inline cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/AbstractProgramTwillRunnable.java]: I think this should be true.
https://github.com/cdapio/cdap/blob/develop/cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/remote/RemoteExecutionJobMain.java#L142

**Comment by albertshau** [comment]
merging without unit test build since this only affects distributed program runs, and to unblock integration testing

**Pull request merged**